addons/sales_team/models/res_users.py

```diff
@@ -7,6 +7,14 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
+    crm_team_ids = fields.Many2many('crm.team', 'crm_team_member', 'user_id', 'crm_team_id', string='Sales Teams')
+    crm_team_member_ids = fields.One2many('crm.team.member', 'user_id', string='Sales Team Memberships')
+    # redefinition of the field defined in sales_team. The field is now computed
+    # based on the new modeling introduced in this module. It is stored to avoid
+    # breaking the member_ids inverse field. As the relationship between users
+    # and sales team is a one2many / many2one relationship we take the first of
+    # the team.user record to find the user's sales team.
```

Comment:
team.user? You mean crm.team.member?

Also, does `related='crm_team_member_ids.crm_team_id'` actually work?
What happens if there are multiple records? It takes the first one automatically?

A bit curious here as I have rarely seen that kind of related definition.
Usually we would do a real stored computed field with an actual compute method for that kind of behavior.

Reply:
It is changed afterwards. This is just a mov.